### ISSUE TITLE
"make test" broken

### ISSUE THREAD

**Issue by a-m-s** [MycroftAI/mimic#41]
`make -k test` fails with this output:

```
Makefile:79: warning: overriding recipe for target 'multi_thread'
Makefile:76: warning: ignoring old recipe for target 'multi_thread'
make[1]: *** No rule to make target 'test'.
Makefile:130: recipe for target 'test' failed
make: *** [test] Error 2
```

**Comment by forslund** [comment]
I'm not sure make test target is implemented at all. What would you expect the test to do?

Is it mentioned in some of our documentation?

**Comment by a-m-s** [comment]
It's right there in the make file.

I'd expect it to check the build is good, whatever that means for the project. This is not an unusual feature.

If the correct fix is to just remove the vestigial testing cruft then so be it.

**Comment by forslund** [comment]
Right you are! That's what I get for browsing the code on my phone. I'll get right on it, thank you for reporting.

**Comment by forslund** [comment]
This should be resolved in PR #46. Now make test will build the tests and execute them.
